<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsWidget</class>
 <widget class="QWidget" name="DatabaseSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,5,1">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="transformRoundsSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="transformBenchmarkButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Benchmark</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="dbNameLabel">
          <property name="text">
           <string>Database name:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="historyMaxSizeCheckBox">
          <property name="text">
           <string>Max. history size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="transformRoundsLabel">
          <property name="text">
           <string>Transform rounds:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="historyMaxItemsCheckBox">
          <property name="text">
           <string>Max. history items:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="dbNameEdit"/>
        </item>
        <item row="6" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="historyMaxItemsSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>2000000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="defaultUsernameLabel">
          <property name="text">
           <string>Default username:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="dbDescriptionEdit"/>
        </item>
        <item row="7" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="historyMaxSizeSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string> MiB</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="recycleBinEnabledCheckBox">
          <property name="text">
           <string>Use recycle bin</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="defaultUsernameEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="dbDescriptionLabel">
          <property name="text">
           <string>Database description:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="AlgorithmComboBox">
          <item>
           <property name="text">
            <string>AES:  256 Bit   (default)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Twofish:  256 Bit</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="AlgorithmLabel">
          <property name="text">
           <string>Algorithm:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbNameEdit</tabstop>
  <tabstop>dbDescriptionEdit</tabstop>
  <tabstop>transformRoundsSpinBox</tabstop>
  <tabstop>transformBenchmarkButton</tabstop>
  <tabstop>defaultUsernameEdit</tabstop>
  <tabstop>recycleBinEnabledCheckBox</tabstop>
  <tabstop>historyMaxItemsCheckBox</tabstop>
  <tabstop>historyMaxItemsSpinBox</tabstop>
  <tabstop>historyMaxSizeCheckBox</tabstop>
  <tabstop>historyMaxSizeSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
